tests/csvwriter.py

```diff
@@ -5,66 +5,74 @@
 from input_output.file_output import create_csv_file
 import time
 import csv
+from input_output.sys_output import fail, passed
 
 # This file is to demonstrate the workings of writing test results to a CSV file.
 # The test itself is a mock and will always pass.
 # The result of the test are written to CSV in the folder output_files/csv/
-from input_output.sys_output import fail, passed
 
-file = 'colorref_smallexample_4_16'
-filename = 'test_graphs/color_refinement/' + file + '.grl'
-graphs = load_graph_list(filename)
+class Csvwriter:
+    def unittest(self):
+        return _unittest()
+
+
+def _unittest():
+    file = 'colorref_smallexample_4_16'
```

Comment:
Dit is de test voor de CSV writer toch? Dus eigenlijk maakt het helemaal niet uit wat je opslaat, als het maar wat opslaat? Dus ik zou alles wat te maken heeft met het GI problem er uithalen, dat kost alleen maar tijd. Als je gewoon een testregel "TEST TEST TEST" kan schrijven naar een csv en het vervolgens uit kan lezen, is de test geslaagd toch?

Reply:
Dat maakt de code ook een stuk korter een leesbaarder namelijk

Reply:
Ik maak er een issue voor aan, dan kunnen we dat afwegen.
ik heb alleen de file omgezet naar functie in deze PR. Aan de inhoud niks aangepast.